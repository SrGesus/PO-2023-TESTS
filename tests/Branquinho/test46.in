2
saved10
0