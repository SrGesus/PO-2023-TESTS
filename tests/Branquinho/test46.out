Menu Principal
1 - Novo
2 - Abrir
0 - Sair
Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro saved10
Menu Principal
1 - Novo
2 - Abrir
0 - Sair
Escolha uma opção: 